src/redux/greeting/greettingSlice.js

```diff
@@ -0,0 +1,30 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+// eslint-disable-next-line consistent-return
+export const fetchGreeting = createAsyncThunk('greeting/fetchGreeting', async () => {
+  try {
+    const response = await fetch('/greetings');
```

Comment:
- [ ] Excellent progress on your implementation! 👍 However, I'd like to highlight a few key points to enhance your code and help you achieve the primary goal of this project: ensuring your front-end app successfully retrieves data from the `API` served by your backend app.

1. **CORS Issue:** Your app currently faces issues fetching data from the backend due to the lack of `CORS` configuration. The backend needs proper `CORS` setup to allow the front-end app to access its resources. I've provided detailed instructions on how to set this up in your backend project to resolve this hurdle.

2. **Fetching Data:** Ensure your `fetch` request uses the correct URL to access the backend API. Since the backend might be running on a different server, make sure to provide the accurate URL. For instance, during my review, I had the backend running on `http://localhost:3000`. By inspecting your API endpoint route, I appended it to the backend server URL (`http://localhost:3000/greetings/random`) within the `fetchGreeting` function. Doing so successfully retrieved the data from the API.

By addressing these points, your front-end app will effectively communicate with the backend API, enabling seamless data retrieval.

Happy coding! 😊🚀